packages/pyolite-kernel/py/piplite/piplite/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""A configurable Python package backed by Pyodide's micropip"""
+from .piplite import install
+
+__version__ = "0.1.0a16"
```

Comment:
Probably we need to add that one to bumpversion so it stays in sync with the other version numbers for now:

https://github.com/jupyterlite/jupyterlite/blob/b58ea7d358cc8b7438147812739b82e1cc43abc8/.bumpversion.cfg#L19